docker-compose.yaml

```diff
@@ -179,17 +178,25 @@ services:
     container_name: resizer
     environment:
       - AWS_ACCESS_KEY_ID=<AWS_ACCESS_KEY>
-      - AWS_REGION=us-west-1
       - AWS_SECRET_ACCESS_KEY=<AWS_SECRET_ACCESS_KEY>
       - S3_URL=http://fake-s3:4569
       - NETWORK=fabric
       - REDIS_CACHE_PORT=6379
       - REDIS_CACHE_URL=redis
       - S3_BUCKET=mra-images
+      - CPU_THRESHOLD=0.9
+      - MEMORY_THRESHOLD=0.8
+      - DISK_THRESHOLD=0.05
+      - LARGE_IMAGE_SIZE=-1
+      - MEDIUM_IMAGE_SIZE=1280
+      - THUMB_IMAGE_SIZE=128
     ports:
       - "83:443"
     links:
       - redis
+      - fake-s3
+    depends_on:
+      - fake-s3      
```

Comment:
May want to include the depends_on flag for other services/database relationships if this is necessary. What does depends_on add here?

Reply:
depends forces the services to be up and running before starting the service that depends on it. You can enforce load order that way. One thing it does not do is guarantee that the service can actually take requests, it only says that the docker image is loaded. But it is a start. We should go through and add depends to the full hierarchy.